src/FSharp.Build/Fsc.fs

```diff
@@ -49,6 +49,7 @@ type public Fsc() as this =
     let mutable otherFlags: string MaybeNull = null
     let mutable outputAssembly: string MaybeNull = null
     let mutable outputRefAssembly: string MaybeNull = null
+    let mutable parallelCompilation: bool option = None
```

Comment:
I was talking to Kevin another day and AFAIU optional bools are for cases when the feature is to be enabled on the older compilers, to track its usage. Hence realsig is like this but most are just bools.

In other words, is this on purpose?

Reply:
Yes, it is.
If users does not set it (neither enabled, nor disabled in project properties), we keep it as None and let compiler apply the default.

Reply:
Alrighty, thanks.